config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -175,6 +177,20 @@ data:
       installFailingReason: LibvirtConnectionFailed
       installFailingMessage: "Could not connect to libvirt host"
 
+
+    # Proxy-enabled clusters
+    - name: ProxyTimeout
```

Comment:
Is this one [retryable](https://gitlab.cee.redhat.com/service/app-interface/-/blob/7b33783eb34219c03b28c2cbccbb1b29d5df2a87/resources/services/hive/integration/hive.hiveconfig.yaml#L27-30)?

Reply:
I'll let @mrbarge answer, but both of those sound like things that need user intervention (so, no, not retryable)

Reply:
For now I'd agree, if it's timing out, it's likely either a VPC configuration is wrong, or the wrong values have been supplied at install time, so likely not retryable. We can let our future experience be the judge on if that's correct or not. :) 

Thanks for the review folks!